var/spack/repos/builtin/packages/launchmon/package.py

```diff
@@ -4,30 +4,30 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack import *
+import os
 
 
-class Launchmon(Package):
+class Launchmon(AutotoolsPackage):
     """Software infrastructure that enables HPC run-time tools to
        co-locate tool daemons with a parallel job."""
     homepage = "https://github.com/LLNL/LaunchMON"
     url = "https://github.com/LLNL/LaunchMON/releases/download/v1.0.2/launchmon-v1.0.2.tar.gz"
+    git      = "https://github.com/llnl/launchmon.git"
 
+    version('master', branch='master')
     version('1.0.2', '8d6ba77a0ec2eff2fde2c5cc8fa7ff7a')
 
     depends_on('autoconf', type='build')
     depends_on('automake', type='build')
     depends_on('libtool', type='build')
+    depends_on('pkg-config', type='build')
     depends_on('libgcrypt')
     depends_on('libgpg-error')
     depends_on("elf", type='link')
     depends_on("boost")
     depends_on("spectrum-mpi", when='arch=ppc64le')
 
-    def install(self, spec, prefix):
-        configure(
-            "--prefix=" + prefix,
-            "--with-bootfabric=cobo",
-            "--with-rm=slurm")
-
-        make()
-        make("install")
+    def setup_environment(self, spack_env, run_env):
+        #os.environ['ACLOCAL_PATH'] += ':' + os.path.join(self.spec['libgcrypt'].prefix, "share", "aclocal")
+        spack_env.prepend_path('ACLOCAL_PATH', os.path.join(
+            self.spec['libgcrypt'].prefix, "share", "aclocal"))
```

Comment:
You can do `self.spec['libgcrypt'].prefix.share.aclocal`, no `os.path.join` necessary.